samples/tab-deeplink-all-features/csharp/README.md

```diff
@@ -0,0 +1,114 @@
+---
+page_type: sample
+description: Microsoft Teams sample app for demonstrating deeplink all features using Tab
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+extensions:
+ contentType: samples
+ createdDate: "26/01/2023 01:38:27 PM"
+urlFragment: officedev-microsoft-teams-samples-tab-deeplink-all-features-csharp
+
+---
+ # DeepLink All Features
+
+ This sample app for demonstrating for deeplink all features using Tab.
+
+- **Interaction with App**
+
+ ![Deeplink-All-Features](DeeplinkAllFeatures/Images/DeeplinkAllFeatures.gif)
+
+ ## Prerequisites
```

Comment:
Make sure that you have verified below points in your PR: 

Add build pipeline for C#/NodeJS samples.
Add deployment pipeline for every new sample like for C#, if sample exist otherwise create it for NodeJS sample.
Deploy the sample manually, test it and add the deployed manifest in your sample folder.
Make sure that you have added the deployed manifest link in your Readme file.
Make sure to link your sample with main/root readme.
Make sure that you have linked the proper Microsoft Documentation with your sample's readme file in "Further reading" section.
Once sample is pushed, make sure to raise a doc bug on Microsoft documentation to link your sample on that page.
Make sure that you have added sample.json with proper links of .gif and metadata.
Make sure to run build pipeline every time you push the new changes in PR, including PR comments fixes and add the screenshot/link with your PR.
Verify that your readme file is visible properly in preview section.

Reply:
Code related guidelines:

Remove all extra spaces and lines from your code.
Make sure that you have added proper summary for each and every methods.
Don't push credentials in your PR.
Verify alignments and correct those.
Don't put any hardcoded value and push those in PR.
Remove unwanted commented code
Use latest framework and tools for your code implementation